README.md

```diff
@@ -6,6 +6,53 @@ Guardian is a Terraform actuation and enforcement tool using GitHub actions.
 
 **This is not an official Google product.**
 
+## Guardian Features
+
+* Terraform Actuation via Plan, Apply, Run, and Admin cli
+  * Show terraform plans and applies, including outputs, in GitHub comments on Pull Requests.
+  * Determine all terraform entrypoints (e.g. where your terraform backend configurations are)
+    in your repository and plan/apply for each entrypoint.
+  * Automatically detect changes and only plan/apply entrypoints that have changed.
+  * Ensures terraform plans are always up to date via native GitHub functionality.
+  * Designed to be a drop-in replacement for terraform enterprise.
+  * Compatible with any version of terraform. 
+  * For more details, see [Developer Workflow](#developer-workflow).
+* IAM Drift Detection via IAM drift cli
+  * Compatible with GCP.
```

Comment:
nit: GCP -> Google Cloud throughout

Avoid short hand if possible, e.g. Google Cloud Organization instead of GCP Org

Reply:
Done